packages/widget/src/components/Routes/RoutesExpanded.tsx

```diff
@@ -112,15 +120,20 @@ export const RoutesExpandedElement = () => {
   }, [emitter, expanded]);
 
   return (
-    <Collapse
+    <RoutesExpandedCollapse
       timeout={timeout.enter}
       in={expanded}
       orientation="horizontal"
       onExited={onExit}
     >
-      <Grow timeout={timeout.enter} in={expanded} mountOnEnter unmountOnExit>
-        <Container enableColorScheme>
-          <ScrollableContainer>
+      <RouteNestedGrow
+        timeout={timeout.enter}
+        in={expanded}
+        mountOnEnter
+        unmountOnExit
+      >
+        <Container enableColorScheme minimumHeight={isLoading}>
+          <ScrollableContainer id="ScrollableContainer">
```

Comment:
Let's add this id attribute here please to be consistent https://github.com/lifinance/widget/blob/main/packages/widget/src/utils/elements.ts#L1

Reply:
I think can remove these - I've accidently left them in from debug the routes positioning so they aren't really needed